timemachine/potentials/nonbonded.py

```diff
@@ -261,8 +261,8 @@ def nonbonded_on_specific_pairs(
     box,
     pairs,
     beta: float,
-    cutoff: Optional[float] = None,
-    w_coords: Optional[Array] = None,  # per-particle 4-d coordinate
+    cutoff=None,
```

Comment:
@mcwitt: I'm going to remove these two typings because its giving me too much grief on my python. I'll add them back me once we migrate to python11

Reply:
OK. Would it also fix to change `Array: TypeAlias = NDArray` to `Array = Any`? The latter is the convention we've adopted in other places, and IIUC it's the `TypeAlias` that's breaking with 3.7.0b0?

Reply:
I think that works actually - I'll just set `Array = Any` instead of remove these annotations.

Reply:
fixed - thanks for noticing this.